fog/ingest/enclave/src/lib.rs

```diff
@@ -60,6 +60,7 @@ pub struct IngestSgxEnclave {
     // Hold a reference counter to the enclave to prevent destruction.
     // This is a proxy object, its lifetime is not in lock-step with the
     // actual enclave.
+    #[allow(dead_code)]
     enclave: Arc<SgxEnclave>,
```

Comment:
I believe renaming with the underscore will eliminate the dead code warning
```suggestion
    _enclave: Arc<SgxEnclave>,
```